sections/featured-product.liquid

```diff
@@ -213,11 +213,7 @@
                 </div>
               {%- when 'share' -%}
                 {% assign share_url = product.selected_variant.url | default: product.url | prepend: request.origin %}
-                {% render 'share-button-block',
-                  block: block,
-                  share_link: share_url
-                %}
-
+                {% render 'share-button-block', block: block, share_link: share_url %}
```

Comment:
I actually didn't realize we named the share block "share-button-block". Not a big deal and obviously not super related to this PR, but if you felt like it I wouldn't renaming that to just `share-button` for consistency.

Reply:
Thanks for including that. Retested and all 3 share button instances still work as expected.